diff --git a/src/Classes/MyRadioEmail.php b/src/Classes/MyRadioEmail.php
index a1ee5190..280330b6 100644
--- a/src/Classes/MyRadioEmail.php
+++ b/src/Classes/MyRadioEmail.php
@@ -163,7 +163,7 @@ class MyRadioEmail extends ServiceAPI
 
     private function getHeader()
     {
-        $headers = ['MIME-Version: 1.0'];
+        $headers = ['MIME-Version: 1.0', 'Content-Transfer-Encoding: quoted-printable'];
 
         if ($this->from !== null) {
             $headers[] = 'From: '.$this->from->getName().' <'.$this->from->getEmail().'>';
@@ -221,7 +221,7 @@ class MyRadioEmail extends ServiceAPI
                     if (!mail(
                         $user->getName() . ' <' . $user->getEmail() . '>',
                         $u_subject,
-                        $u_message,
+                        quoted_printable_encode($u_message),
                         $this->getHeader()
                     )) {
                         continue;

